packages/dnb-eufemia/src/extensions/forms/constants/locales/en-GB.ts

```diff
@@ -120,6 +120,7 @@ export default {
       errorRequired: 'You must enter a national identity number.',
       errorFnr: 'Invalid national identity number.',
       errorDnr: 'Invalid D number.',
+      errorBelowAge: 'Must be at least {age} years of age.',
```

Comment:
```suggestion
      ageValidatorError: 'Must be at least {age} years of age.',
```
Maybe this is more in line of how we’ve named the other translations?

Reply:
I'll make the change then.